evalml/tests/dependency_update_check/latest_dependency_versions.txt

```diff
@@ -23,7 +23,7 @@ scikit-optimize==0.8.1
 scipy==1.6.2
 seaborn==0.11.1
 shap==0.39.0
-sktime==0.5.3
+sktime==0.6.0
```

Comment:
is this dependency change for this PR?

Reply:
oo this was for the dependency test right? I just merged in the PR so this isn't needed anymore 😄

Reply:
No this was unrelated but needed to pass the dependency check, sktime had a release a few hours ago and it doesn't break any of our code but I thought I should include it for bug fixes etc.

Reply:
^^Ignore this haha I didn't see your next comment